iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/visitors/LocationShifter.java

```diff
@@ -207,6 +214,20 @@ public static MarkedYamlEngineException shiftMarkedYamlException(HelmInputFileCo
     return exception;
   }
 
+  private static boolean isTextRangeValid(TextRange mixedTextRange, HelmInputFileContext currentCtx) {
+    try {
+      currentCtx.inputFile.newRange(
+        mixedTextRange.start().line(),
+        mixedTextRange.start().lineOffset(),
+        mixedTextRange.end().line(),
+        mixedTextRange.end().lineOffset());
+      return true;
+    } catch (IllegalArgumentException e) {
+      // the text range is invalid
```

Comment:
Should we log these cases? To be able to reproduce and fix them eventually. Just by inaccurate ranges it will be hard to detect.

Reply:
IMO, no. We don't have any process in place to review all logs. We do this in our code all the best to highlight the best element. We will see when reviewing Peachee jobs that some locations are strange.